README.md

```diff
@@ -175,7 +175,7 @@ While quite outdated at this point, [this original blog post about Pi-hole](http
 -----
 
 ## Coverage
-- [Lifehacker: Turn A Raspberry Pi Into An Ad Blocker With A Single Command](https://www.lifehacker.com.au/2015/02/turn-a-raspberry-pi-into-an-ad-blocker-with-a-single-command/) (Feburary, 2015)
+- [Lifehacker: Turn A Raspberry Pi Into An Ad Blocker With A Single Command](https://www.lifehacker.com.au/2015/02/turn-a-raspberry-pi-into-an-ad-blocker-with-a-single-command/) (February, 2015)
```

Comment:
@DL6ER maybe these should be updated from the current docs page? https://github.com/pi-hole/docs/blob/master/docs/main/coverage.md

Reply:
Just take the whole Coverage section out. Link it to docs. The README is bloated anyways.